tests/backup/backup_nfs_test.go

```diff
@@ -203,3 +204,211 @@ var _ = Describe("{DeleteNfsExecutorPodWhileBackupAndRestoreInProgress}", func()
+			multipleNSScheduledBackup, err = CreateScheduleBackupWithValidation(ctx, multipleNSScheduleName, SourceClusterName, bkpLocationName, backupLocationUID, scheduledAppContexts, make(map[string]string), orgID, "", "", "", "", periodicSchedulePolicyName, periodicSchedulePolicyUid)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation and validation of schedule backup with schedule name [%s]", multipleNSScheduleName))
+			err = suspendBackupSchedule(multipleNSScheduleName, periodicSchedulePolicyName, orgID, ctx)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Suspending Backup Schedule [%s]", multipleNSScheduleName))
+			backupNames = append(backupNames, multipleNSScheduledBackup)
+			scheduleNames = append(scheduleNames, multipleNSScheduleName)
+		})
+
+		Step("Taking a custom resource backup of all namespaces", func() {
+			log.InfoD("Taking a custom resource backup of all namespaces")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			customResourceBackupName = fmt.Sprintf("%s-custom-resource-%v", BackupNamePrefix, RandomString(5))
+			appContextsToBackup := FilterAppContextsByNamespace(scheduledAppContexts, appNamespaces)
+			err = CreateBackupWithCustomResourceTypeWithValidation(ctx, customResourceBackupName, SourceClusterName, bkpLocationName, backupLocationUID, appContextsToBackup, []string{"PersistentVolumeClaim"}, nil, orgID, clusterUid, "", "", "", "")
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Creation of custom resource backup - %s", customResourceBackupName))
+			backupNames = append(backupNames, customResourceBackupName)
+		})
+
+		Step("Remove the Cloud Snapshots from the NFS backup location for all the backups", func() {
+			log.InfoD("Remove the Cloud Snapshots from the NFS backup location for all the backups")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			for _, backupName := range backupNames {
+				backupUID, err := Inst().Backup.GetBackupUID(ctx, backupName, orgID)
+				log.FailOnError(err, fmt.Sprintf("Getting UID for backup %v", backupName))
+				backupInspectRequest := &api.BackupInspectRequest{
+					Name:  backupName,
+					Uid:   backupUID,
+					OrgId: orgID,
+				}
+				resp, err := Inst().Backup.InspectBackup(ctx, backupInspectRequest)
+				log.FailOnError(err, fmt.Sprintf("error inspecting backup %v", backupName))
+				// the volume path should be supplied instead of the metadataPath
+				//volumePath := resp.Backup.Volumes
+				//for _, ele := range volumePath {
+				//	bkpID := ele.BackupId
+				//	parts := strings.Split(bkpID, "/")
+				//	subparts := strings.Split(parts[1], "-")
+				//	currentSnapShotPath := parts[0] + "/" + subparts[0] + "/" + subparts[0] + "-" + subparts[1]
+				//	RemoveCloudSnapshotsFromNFSLocation(currentSnapShotPath)
+				//}
+				//the below code is a work around
+				metadataPath := globalBucket + "/" + resp.Backup.BackupPath
```

Comment:
We can uncomment the above part  and when the bug is fixed we will only have to delete lines 349-351
Please add the bug number also for reference

Reply:
Done.